package.json

```diff
@@ -100,6 +101,7 @@
     "bindings": "^1.3.0",
     "buffers": "^0.1.1",
     "classnames": "^2.2.5",
+    "component-emitter": "^1.2.1",
```

Comment:
What is this for?

Reply:
The decoder uses it.
It's the same emitter used by socket.io internally